scenarios/kubernetes_bazel.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Runs bazel build/test for current repo."""
+
+import argparse
+import os
+import subprocess
+import sys
+
+ORIG_CWD = os.getcwd()
+
+def test_infra(*paths):
+    """Return path relative to root of test-infra repo."""
+    return os.path.join(ORIG_CWD, os.path.dirname(__file__), '..', *paths)
+
+
+def check_output(*cmd):
+    """Log and run the command, return output, raising on errors."""
+    print >>sys.stderr, 'Run:', cmd
+    return subprocess.check_output(cmd)
+
+
+def check(*cmd):
+    """Log and run the command, raising on errors."""
+    print >>sys.stderr, 'Run:', cmd
+    subprocess.check_call(cmd)
+
+
+def main(build, test, push):
+    """Trigger a bazel build/test run, and upload results."""
+    os.environ['TEST_TMPDIR'] = '/root/.cache/bazel'
```

Comment:
maybe set this in the prow config instead? I think that's the pattern we want to move to, since the prow config is where we mount the volume. (see e.g. https://github.com/kubernetes/test-infra/blob/d67c33daddccb78f74ca7be4bfe8c5fb2893de06/prow/config.yaml#L465-L467)